src/SWP/Bundle/ContentListBundle/Doctrine/ORM/ContentListRepository.php

```diff
@@ -33,4 +33,12 @@ public function findByType(string $type): array
             ->getResult()
         ;
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function flush()
```

Comment:
Yes, this should be removed - as `flush()` is added to `EntityRepository`. And i'm against removing it from there. As `add` do almost this same as `persist` and `flush` and exists there. It's very convenient to have those methods in repository - it's in most of the cases one service less to inject. And gives much more controll than simple `add`.